### DISCUSSION TITLE
Version 2.40 also includes Token Introspection

### DISCUSSION THREAD

**Discussion by supercairos** [dexidp/dex#3575] [General]
Hi,

I belive the Token Introspection is also bundled into the [v2.40.0](https://github.com/dexidp/dex/releases/tag/v2.40.0) version but doesn't seams to be mentioned:
https://github.com/dexidp/dex/issues/3387
https://github.com/dexidp/dex/commit/87553087591db31de96953fed038b2447924b3af

cc @nabokihms

**Comment by nabokihms** [comment]
Thanks, added it to the release message. We use GitHub for autogenerated changelogs. Probably it is some kind of a bug idk
